.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
nit: do all of these settings need to be in the repo settings? not all users do things the same way and you can achieve this for all of your projects by putting the same information in your user settings

Reply:
The reason for keeping them here is: If someone wants to contribute to the project, then they don't have to worry about indentation for this project. It becomes easy to review PRs and improves productivity for the reviewer and the contributor.
The `.vscode/` dir is set in .`npmignore` so nothing from this dir is published.

Reply:
it makes sense for the indentation (though you might consider something like [editorconfig](https://editorconfig.org/) to support a broader set of tools). it's really the other settings. for instance, i actually like to see the node modules in the file tree but with these settings i don't have the choice because the repository itself has shut it off.

not a huge deal either way